lib/sync_checker/formats/publication_check.rb

```diff
@@ -64,7 +64,7 @@ def filter_documents_for_locale(edition, locale)
         locales_to_filter = Array(locale.to_s)
         locales_to_filter << "" if locale.to_s == I18n.default_locale.to_s
         locale_attachments = all_attachments.select do |attachment|
-          attachment.is_a?(FileAttachment) ||
+          !attachment.is_a?(FileAttachment) &&
```

Comment:
Should we make this more explicit and say `attachment.is_a?(HtmlAttachment)` as they are all we are interested in here?